app/controllers/roles_controller.rb

```diff
@@ -40,7 +40,7 @@ def create
             email: user.email
           }
           @role.user = user
-          if @role.save
+          if @role.save!
```

Comment:
I think you do want `save` instead of `save!`.  The former returns true/false while the latter raises an exception

Reply:
good catch